movies/src/components/Movie.js

```diff
@@ -0,0 +1,35 @@
+//movie
+import React, {Component} from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { getMoviesById } from '../actions';
+
+class Movie extends Component {
+  constructor(props) {
+  super(props);
```

Comment:
No need to pass `props` to the constructor and `super` since you aren't using `props` inside the constructor. Also, indent `super`.